industry_restaurant/data/resource_calendar_data.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo noupdate="1">
+    <record id="resource.resource_calendar_std" model="resource.calendar">
```

Comment:
What happens if this record was deleted before? Without anything, it'll raise an error. You need to add `forcecreate`

Also, from the spec, I think this should be in demo, right?